site/public/js/testcase-output.js

```diff
@@ -0,0 +1,102 @@
+// Map to store the XMLHttpRequest object returned by getJSON when getting the
+// output to display in loadTestcaseOutput()
+// Key: orig_div_name   Value: XMLHttpRequest object
+const output_xhrs = new Map();
+
+/**
+ * Collapses Test Case Output.
+ *
+ * @param {string} div_name - id of div containing the output of the test case
+ * @param {string} index - index of test case
+ * @param {object} loadingTools - loading tools used by span containing the text that
+ *                                correspond to the state of the test case (show, hide, loading)
+ */
+function CollapseTestcaseOutput(div_name, index, loadingTools) {
+    $(`show_char_${index}`).toggle();
```

Comment:
Is this supposed to be selecting a class/id?

Reply:
Yes.  The `$(selector)` is the jQuery Selector Syntax for selecting HTML elements.